### ISSUE TITLE
Reduce accessibility modifiers as much as possible

### ISSUE THREAD

**Issue by ralexstokes** [ethereum/eth2.0-specs#1151]
Some functions in the deposit contract are marked `@public` when they have no reason to be (other than implementation details of the vyper stack, i.e. they are easier to test when marked `@public`).

I suggest in the spirit of keeping this contract as simple and small as possible, we reduce the size of the public API by remove the `@public` decorators where we can.

At the moment that is:

```python
@public
@constant
def to_little_endian_64(value: uint256) -> bytes[8]:
```

assuming we also address #34.

**Comment by JustinDrake** [comment]
As you mention, removing `@public` on `to_little_endian_64` breaks the tests.

Please open a PR ASAP if you feel strongly about this :) It will be harder to make changes after spec freeze, and after formal verification of the deposit contract.

**Comment by JustinDrake** [comment]
Fixed in #1152 :)
